requirements.txt

```diff
@@ -11,9 +11,9 @@ applicationinsights
 prompt_toolkit
 rope
 tox
-pyyaml>=4.1,<=4.2b4
+pyyaml==5.4
```

Comment:
>== [](start = 6, length = 2)

why == and not >=?

Reply:
didn't want to extrapolate beyond the requirements